### PR TITLE
#82: Moved status api out of versioning routes

### PR DESCRIPTION
**Pull request by ulin-evgeny**

### CODE REVIEW

File: tests/StatusTest.php [hunk 1]
```diff
@@ -10,7 +10,7 @@ class StatusTest extends TestCase
 {
     public function testStatusOk()
     {
-        $response = $this->json('get', '/status');
+        $response = $this->json('get', '/status', [], [], 0, false);
```
Comment (DenTray) [inline tests/StatusTest.php]:
```suggestion
        $response = $this->get('/status');
```

File: tests/StatusTest.php [hunk 2]
```diff
@@ -23,7 +23,7 @@ public function testStatusServiceUnavailable()
         DB::shouldReceive('connection')->andReturn($connection);
         DB::shouldReceive('getPdo')->andThrow(Exception::class);
 
-        $response = $this->json('get', '/status');
+        $response = $this->json('get', '/status', [], [], 0, false);
```
Comment (DenTray) [inline tests/StatusTest.php]:
```suggestion
        $response = $this->get('/status');
```

File: tests/TestCase.php [hunk 3]
```diff
@@ -67,10 +67,13 @@ public function call($method, $uri, $parameters = [], $cookies = [], $files = []
         return parent::call($method, $uri, $parameters, $cookies, $files, $server, $content);
     }
 
-    public function json($method, $uri, array $data = [], array $headers = [], $options = 0, ?VersionEnum $apiVersion = null): TestResponse
+    public function json($method, $uri, array $data = [], array $headers = [], $options = 0, VersionEnum|bool|null $apiVersion = null): TestResponse
```
Comment (DenTray) [inline tests/TestCase.php]: please use `get` method for the status tests instead of extending `json` method

**Review by DenTray — CHANGES_REQUESTED** [review]

File: tests/TestCase.php (hunk 3 above)
Comment (ulin-evgeny) [reply tests/TestCase.php]: Done.

File: tests/StatusTest.php (hunk 2 above)
Comment (ulin-evgeny) [reply tests/StatusTest.php]: Done.

File: tests/StatusTest.php (hunk 1 above)
Comment (ulin-evgeny) [reply tests/StatusTest.php]: Done.

**Review by DenTray — APPROVED** [review]

**Pull request merged**